_scripts/pm2-all.sh

```diff
@@ -21,12 +21,21 @@ then
 fi
 
 echo "${COMMAND} fxa services..."
-if yarn workspaces foreach --topological-dev --verbose --exclude fxa-dev-launcher --exclude fxa run "$COMMAND" > artifacts/start.log;
+if ! yarn workspaces foreach --topological-dev --verbose --exclude fxa-dev-launcher --exclude fxa run "$COMMAND" > artifacts/start.log;
+then
+  echo -e "\n###########################################################\n"
+  echo "# fxa couldn't start. see ./artifacts/start.log for details"
+  echo -e "\n###########################################################\n"
+  exit 1
+fi
+
+echo "${COMMAND} nx projects..."
+if nx run-many -t pm2-"$COMMAND" > artifacts/start.log;
```

Comment:
Curious why we're adding `nx run-many` but keeping `yarn workspaces foreach` here?